### PR TITLE
support tmux (sixel enabled)

### PR DESCRIPTION
**Pull request by mattn**
Recently, tmux support DRCS sixel. This change enable tmux if it is enabled.

### CODE REVIEW

**Comment by rs** [comment]
Could you please add a few comments in the code to explain what is doing what? I'm particularly interesting about the HasPrefix line.

**Comment by mattn** [comment]
Added comment

File: term/sixel.go [hunk 1]
```diff
@@ -40,6 +39,13 @@ func checkSixel() bool {
 	if err != nil {
 		return false
 	}
+	// Check tmux (sixel enabled)
+	// See: CSI Ps c, Ps = 4 (Sixel graphics)
+	// https://invisible-island.net/xterm/ctlseqs/ctlseqs.html
+	// NOTE: tmux does not return VT name.
+	if bytes.HasPrefix(b[:n], []byte("\x1b[?1;2;4c")) {
```
Comment (rs) [inline term/sixel.go:46]: why not adding this prefix to the `supportedTerminals` list?

Comment (mattn) [reply term/sixel.go:46]: `1;2;` is not terminal name like `62` (vt200) or `63` (vt300).

**Comment by rs** [comment]
Thx

**Pull request merged**